multivac/gather_job_data.py

```diff
@@ -83,13 +83,33 @@ def __init__(self, cli_args):
         self.gathered_data = dict()
         self.latest_n: int = cli_args.latest
         self.watch_failure = args.watch_failure
+        self.since_seconds = None
+
+        since: str = cli_args.since
+        if since and len(since) > 1:
+            wrong_usage_message = 'Wrong \'--since\' option: got wrong %s. ' \
+                                  'Usage: NN[d|h], example: 42d.'
+            if since.endswith('d'):
+                try:
+                    self.since_seconds = int(since.rstrip('d')) * 86400
+                except ValueError:
+                    print(wrong_usage_message % 'number')
```

Comment:
please use f-strings, we're already using them in this repo

Reply:
Fixed with `.format()`